diff --git a/main.go b/main.go
index 579a304..752b192 100644
--- a/main.go
+++ b/main.go
@@ -11,7 +11,7 @@ func main() {

        byt, _ := ioutil.ReadFile("example.diff")
        diff, _ := diffparser.Parse(string(byt))
-
+       fmt.Println("hello")
        //现在，您可以从差异中获取文件的一部分，
        for k := range diff.Files {
                //获取diff 头


@@ -20,10 +20,9 @@ func main() {
                        hunk := file.Hunks[k1]
                        // new and old ranges in the hunk
                        newRange := hunk.NewRange
-
+                       fmt.Println("hello", newRange)
                        // and lines in the ranges.
-                       line := newRange.Lines[0]
-                       fmt.Println(line)
+
                }
        }
 }


