diff --git a/main.go b/main.go
index 7905807..41e7f16 100644
--- a/main.go
+++ b/main.go
@@ -1,5 +1,7 @@
 package main
 
-func main() {
+import "fmt"
 
+func main() {
+       fmt.Println("main")
 }
diff --git a/test/test1.go b/test/test1.go
index 8001ad7..065789c 100644
--- a/test/test1.go
+++ b/test/test1.go
@@ -1,5 +1,7 @@
 package test
 
-func test1() {
+import "fmt"
 
+func test1() {
+       fmt.Println("test1")
 }
diff --git a/test/test2.go b/test/test2.go
index b1880a5..cde2083 100644
--- a/test/test2.go
+++ b/test/test2.go
@@ -1,5 +1,7 @@
 package test
 
-func test2() {
+import "fmt"
 
+func test2() {
+       fmt.Println("test1")
 }
~
~