diff --git a/main.go b/main.go
index c12fd42..811810d 100644
--- a/main.go
+++ b/main.go
@@ -8,12 +8,12 @@ import (

 // error handling left out for brevity
 func main() {
+
+       byt, _ := ioutil.ReadFile("example.diff")
+       diff, _ := diffparser.Parse(string(byt))
        fmt.Println("hello git diff")
        fmt.Println("hello git diff")
        fmt.Println("hello git diff")
-       byt, _ := ioutil.ReadFile("example.diff")
-       diff, _ := diffparser.Parse(string(byt))
-
        // You now have a slice of files from the diff,
        file := diff.Files[0]
        // diff hunks in the file,
diff --git a/test/test.go b/test/test.go
index 56e5404..2675738 100644
--- a/test/test.go
+++ b/test/test.go
@@ -1 +1,7 @@
 package test
+
+import "fmt"
+
+func test() {
+       fmt.Println("hello test")
+}